components/scream/src/physics/shoc/atmosphere_macrophysics.cpp

```diff
@@ -281,9 +283,12 @@ void SHOCMacrophysics::initialize_impl (const util::TimeStamp& /* t0 */)
   auto tke_copy    = m_buffer.tke_copy;
   auto shoc_ql2    = m_buffer.shoc_ql2;
 
-  shoc_preprocess.set_variables(m_num_cols,m_num_levs,m_num_tracers,m_cell_area,
-                                T_mid,z_int,z_mid,p_mid,pseudo_density,omega,phis,surf_sens_flux,surf_latent_flux,
-                                surf_mom_flux,qv,qc,tke,tke_copy,cell_length,
+  // For now, set z_int(i,nlevs) = z_surf = 0
+  const Real z_surf = 0.0;
```

Comment:
You already have a `z_surf` member of this class. This will generate a warning. Either delete the class member, or use that (instead of the temporary).

Reply:
Nvm, I think the member was inside SHOCPreprocess.